manifests/config.pp

```diff
@@ -35,7 +35,10 @@
     })
   }
 
-  ensure_resource('file', $::jenkins::localstatedir, $dir_params)
+  if $::jenkins::manage_localstatedir {
+    ensure_resource('file', $::jenkins::localstatedir, $dir_params)
+  }
+
```

Comment:
I will update the PR so plugin dir and job dir are in the same conditional block as localstatedir. It'll be in a couple of hours.